react/index.tsx

```diff
@@ -1,88 +1,117 @@
 import React, { Fragment, FC, useRef, useEffect, useCallback } from 'react'
 import { path } from 'ramda'
+import debounce from 'lodash/debounce'
```

Comment:
Use `debounce` from `debounce` package instead.

Reply:
q

Reply:
`import debounce from 'debounce'` this package is slimmer

Reply:
![image](https://user-images.githubusercontent.com/284515/69181042-260af280-0aed-11ea-8283-2c6a093026aa.png)
![image](https://user-images.githubusercontent.com/284515/69181048-286d4c80-0aed-11ea-86ce-4fbd45c8a0b5.png)

Reply:
Done.